.github/compatibility-test-matrices/release-v4.2.x/incentivized-transfer-chain-a.json

```diff
@@ -1,16 +1 @@
-{
-  "chain-a": ["release-v4.2.x"],
-  "chain-b": ["release-v4.2.x", "v6.1.0", "v5.2.0", "v4.1.1"],
-  "entrypoint": ["TestIncentivizedTransferTestSuite"],
-  "test": [
-    "TestMsgPayPacketFee_AsyncSingleSender_Succeeds",
-    "TestMsgPayPacketFee_InvalidReceiverAccount",
-    "TestMultiMsg_MsgPayPacketFeeSingleSender",
-    "TestMsgPayPacketFee_SingleSender_TimesOut",
-    "TestPayPacketFeeAsync_SingleSender_NoCounterPartyAddress",
-    "TestMsgPayPacketFee_AsyncMultipleSenders_Succeeds"
-  ],
-  "relayer-type": ["rly"],
-  "chain-binary": ["simd"],
-  "chain-image": ["ghcr.io/cosmos/ibc-go-simd"]
-}
+{ "chain-a": [ "release-v4.2.x" ], "chain-b": [ "release-v4.2.x", "v6.1.0", "v5.2.0", "v4.1.1", "v5.3.0" ], "entrypoint": [ "TestIncentivizedTransferTestSuite" ], "test": [ "TestMsgPayPacketFee_AsyncSingleSender_Succeeds", "TestMsgPayPacketFee_InvalidReceiverAccount", "TestMultiMsg_MsgPayPacketFeeSingleSender", "TestMsgPayPacketFee_SingleSender_TimesOut", "TestPayPacketFeeAsync_SingleSender_NoCounterPartyAddress", "TestMsgPayPacketFee_AsyncMultipleSenders_Succeeds" ], "relayer-type": [ "rly" ], "chain-binary": [ "simd" ], "chain-image": [ "ghcr.io/cosmos/ibc-go-simd" ] }
```

Comment:
if we change `release-script.sh` to output the file with:

```bash
echo $updated_json | jq '.' > $file
```

instead of:

```
echo $updated_json > $file
```

these will instead be pretty printed (and easier for reviewing!)